common/protob/messages-ethereum-eip712-onekey.proto

```
@@ -88,3 +88,37 @@ message EthereumTypedDataValueAckOneKey {
+ * @next Failure
+ */
+message EthereumGnosisSafeTxRequest {
+}
+/**
+ * Request: the Gnosis Safe transaction data
+ * @start
+ * @next EthereumTypedDataSignatureOneKey
+ * @next Failure
+ */
+message EthereumGnosisSafeTxAck {
+    required string to = 1;
```

Buf (1.59.0)

Field named "to" should not be required.

(FIELD_NOT_REQUIRED)